tools/vagrant/README.md

```diff
@@ -42,41 +43,63 @@ wsk action invoke /whisk.system/samples/echo -p message hello --blocking --resul
     "message": "hello"
 }
 ```
-**Tip:** The very first build may take 10 minutes or more depending on network speed. 
-If there are any build failures, it might be due to network timeouts, run `../resume_build` on the host.
-
+**Tip:** The very first build may take 30 minutes or more depending on network speed. 
+If there are any build failures, it might be due to network timeouts, to recover follow the manual
+process to build and deploy in [ansible/README.md](../../ansible/README.md)
 
 **Tip:** By default, each `docker` command will timeout after 840 seconds (14 minutes). If you're on a really slow connection,
 this might be too short. You can modify the timeout value in [docker.gradle](../../../docker.gradle#L22) as needed.
 
+### Using CLI from outside the VM
+You can use the CLI from the host machine as well as from inside the virtual machine.
+The IP address of the virtual machine accesible from outside is `192.168.33.13`.
+If you start another Vagrant VM take into account that the IP address will conflict, use `vagrant suspend` before starting another VM with the same IP address.
+
+From your _host_, configure `wsk` to use your Vagrant-hosted OpenWhisk deployment and run the "echo" action again to test.
+```
+# Set your OpenWhisk Namespace and Authorization Key.
+  wsk property set --apihost 192.168.33.13 --namespace guest --auth `vagrant ssh -- cat openwhisk/ansible/files/auth.guest`
```

Comment:
@csantanapr This assumes I have the `wsk` CLI on my system path, but doesn't really state that I need to do that.  I suggest either mentioning that it should be on your path, or include the relative path to the CLI. I think doing the former is probably easier...